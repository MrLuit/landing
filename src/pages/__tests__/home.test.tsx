import React from 'react';
import renderer from 'react-test-renderer';

import HomePage from '@/pages/index';

describe('Home Page', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<HomePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
