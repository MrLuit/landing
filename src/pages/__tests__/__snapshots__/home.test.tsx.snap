// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders correctly 1`] = `
<div
  className="sc-AxjAm sc-AxirZ keAExr"
  width="100%"
>
  <div
    className="sc-AxjAm bgAmkT"
  >
    <div
      className="sc-AxjAm hPWGaW"
      width={
        Object {
          "lg": "70%",
          "md": "80%",
          "sm": "90%",
        }
      }
    >
      <div
        className="sc-AxjAm sc-AxirZ jcPbhY"
      >
        <h2
          className="sc-AxjAm hbdlIJ"
          fontFamily="body"
        >
          You're in the right place
        </h2>
        <div
          className="sc-AxjAm bgQMNa"
          fontFamily="body"
        >
          It's Time To Use MyCrypto For Your Crypto
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-AxjAm bgAmkT"
  >
    <div
      className="sc-AxjAm hPWGaW"
      width={
        Object {
          "lg": "70%",
          "md": "80%",
          "sm": "90%",
        }
      }
    >
      <div
        className="sc-AxjAm sc-AxirZ jcPbhY"
      >
        <h2
          className="sc-AxjAm hbdlIJ"
          fontFamily="body"
        >
          Fully Compatible
        </h2>
        <div
          className="sc-AxjAm bgQMNa"
          fontFamily="body"
        >
          Use with your favorite hardware and software wallets
        </div>
        <div
          className="sc-AxjAm sc-AxirZ glDYrY"
        >
          <div
            className="sc-AxjAm fNmBSC sc-AxiKw hiYNpo"
          >
            <a
              className="css-vurnku"
              href="add-account/web3"
              rel="noreferrer"
              target="_blank"
            >
              <div
                className="sc-AxjAm fWJZRy"
              >
                <test-file-stub
                  height="100%"
                  width="auto"
                />
              </div>
              <div
                className="sc-AxjAm bgQMNa"
                fontFamily="body"
              >
                Metamask
              </div>
            </a>
          </div>
          <div
            className="sc-AxjAm fNmBSC sc-AxiKw hiYNpo"
          >
            <a
              className="css-vurnku"
              href="add-account/ledger_nano_s"
              rel="noreferrer"
              target="_blank"
            >
              <div
                className="sc-AxjAm fWJZRy"
              >
                <test-file-stub
                  height="100%"
                  width="auto"
                />
              </div>
              <div
                className="sc-AxjAm bgQMNa"
                fontFamily="body"
              >
                Ledger
              </div>
            </a>
          </div>
          <div
            className="sc-AxjAm fNmBSC sc-AxiKw hiYNpo"
          >
            <a
              className="css-vurnku"
              href="add-account/trezor"
              rel="noreferrer"
              target="_blank"
            >
              <div
                className="sc-AxjAm fWJZRy"
              >
                <test-file-stub
                  height="100%"
                  width="auto"
                />
              </div>
              <div
                className="sc-AxjAm bgQMNa"
                fontFamily="body"
              >
                Trezor
              </div>
            </a>
          </div>
          <div
            className="sc-AxjAm fNmBSC sc-AxiKw hiYNpo"
          >
            <a
              className="css-vurnku"
              href="add-account/walletconnect"
              rel="noreferrer"
              target="_blank"
            >
              <div
                className="sc-AxjAm fWJZRy"
              >
                <test-file-stub
                  height="100%"
                  width="auto"
                />
              </div>
              <div
                className="sc-AxjAm bgQMNa"
                fontFamily="body"
              >
                WalletConnect
              </div>
            </a>
          </div>
          <div
            className="sc-AxjAm fNmBSC sc-AxiKw hiYNpo"
          >
            <a
              className="css-vurnku"
              href="add-account/web3"
              rel="noreferrer"
              target="_blank"
            >
              <div
                className="sc-AxjAm fWJZRy"
              >
                <test-file-stub
                  height="100%"
                  width="auto"
                />
              </div>
              <div
                className="sc-AxjAm bgQMNa"
                fontFamily="body"
              >
                Frame
              </div>
            </a>
          </div>
          <div
            className="sc-AxjAm fNmBSC sc-AxiKw hiYNpo"
          >
            <a
              className="css-vurnku"
              href="add-account/web3"
              rel="noreferrer"
              target="_blank"
            >
              <div
                className="sc-AxjAm fWJZRy"
              >
                <test-file-stub
                  height="100%"
                  width="auto"
                />
              </div>
              <div
                className="sc-AxjAm bgQMNa"
                fontFamily="body"
              >
                Trust
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-AxjAm bgAmkT"
  >
    <div
      className="sc-AxjAm hPWGaW"
      width={
        Object {
          "lg": "70%",
          "md": "80%",
          "sm": "90%",
        }
      }
    >
      <div
        className="sc-AxjAm sc-AxirZ jcPbhY"
      >
        <h2
          className="sc-AxjAm hbdlIJ"
          fontFamily="body"
        >
          Managing and storing your assets has never been easier
        </h2>
        <div
          className="sc-AxjAm bgQMNa"
          fontFamily="body"
        >
          The MyCrypto apps put the power in your hands.
        </div>
        <div
          className="sc-AxjAm bgAmkT"
        >
          <h2
            className="sc-AxjAm hbdlIJ"
            fontFamily="body"
          >
            Manage Your Wallets
          </h2>
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            MyCrypto allows you to create, import, and manage all the wallets you'll ever need.
          </div>
          <div
            className="sc-AxjAm bgAmkT"
          >
            <test-file-stub
              height="100%"
              width="auto"
            />
          </div>
        </div>
        <div
          className="sc-AxjAm bgAmkT"
        >
          <h2
            className="sc-AxjAm hbdlIJ"
            fontFamily="body"
          >
            View Your Dashboard
          </h2>
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            See a detailed breakdown of all your wallets, tokens, and transactions.
          </div>
          <div
            className="sc-AxjAm ctWtTc"
            width="auto"
          >
            <test-file-stub
              height="100%"
              width="auto"
            />
          </div>
        </div>
        <div
          className="sc-AxjAm bgAmkT"
        >
          <h2
            className="sc-AxjAm hbdlIJ"
            fontFamily="body"
          >
            Control Your Crypto
          </h2>
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            Manage your information and your cryptocurrency easily and without fear.
          </div>
          <div
            className="sc-AxjAm bgAmkT"
          >
            <test-file-stub
              height="100%"
              width="auto"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-AxjAm hFuQWu"
  >
    <div
      className="sc-AxjAm hPWGaW"
      width={
        Object {
          "lg": "70%",
          "md": "80%",
          "sm": "90%",
        }
      }
    >
      <div
        className="sc-AxjAm sc-AxirZ jkHUSn"
        color="textInvert"
      >
        <h2
          className="sc-AxjAm hbdlIJ"
          fontFamily="body"
        >
          Everythind is Even More Secure with our Desktop App
        </h2>
        <div
          className="sc-AxjAm bgQMNa"
          fontFamily="body"
        >
          Keep your keys out of the browser with the MyCrypto Desktop App. You get more access to your funds, and scammers get less access to you.
        </div>
        <button
          className="sc-AxjAm gCnANf"
        >
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            Download for macOS
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="sc-AxjAm iknhDZ"
  >
    <div
      className="sc-AxjAm hPWGaW"
      width={
        Object {
          "lg": "70%",
          "md": "80%",
          "sm": "90%",
        }
      }
    >
      <div
        className="sc-AxjAm sc-AxirZ jcPbhY"
      >
        <h2
          className="sc-AxjAm hbdlIJ"
          fontFamily="body"
        >
          A Different Way to Keep Your Assets Safe
        </h2>
        <div
          className="sc-AxjAm sc-AxirZ glDYrY"
        >
          <div
            className="sc-AxjAm bgAmkT"
          >
            <div
              className="sc-AxjAm bgAmkT"
            >
              <test-file-stub
                height="100%"
                width="auto"
              />
            </div>
            <div
              className="sc-AxjAm eHeXRk"
              fontFamily="body"
              fontWeight="bold"
            >
              With Banks
            </div>
            <div
              className="sc-AxjAm bgQMNa"
              fontFamily="body"
            >
              They control your account 
              <br />
              They own your info 
              <br />
              They add fees 
              <br />
              They tell you what you can do 
              <br />
            </div>
          </div>
          <div
            className="sc-AxjAm bgAmkT"
          >
            <div
              className="sc-AxjAm bgAmkT"
            >
              <test-file-stub
                height="100%"
                width="auto"
              />
            </div>
            <div
              className="sc-AxjAm eHeXRk"
              fontFamily="body"
              fontWeight="bold"
            >
              With MyCrypto
            </div>
            <div
              className="sc-AxjAm bgQMNa"
              fontFamily="body"
            >
              You control your account 
              <br />
              You own your info 
              <br />
              No fees are added 
              <br />
              You do whatever you want 
              <br />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-AxjAm bgAmkT"
  >
    <div
      className="sc-AxjAm hPWGaW"
      width={
        Object {
          "lg": "70%",
          "md": "80%",
          "sm": "90%",
        }
      }
    >
      <div
        className="sc-AxjAm sc-AxirZ jcPbhY"
      >
        <h2
          className="sc-AxjAm hbdlIJ"
          fontFamily="body"
        >
          Finally Enjoy Peace of Mind
        </h2>
        <h2
          className="sc-AxjAm hbdlIJ"
          fontFamily="body"
        >
          MyCrypto puts your safety & security first.
        </h2>
        <button
          className="sc-AxjAm gCnANf"
        >
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            Get Started on Web
          </div>
        </button>
        <button
          className="sc-AxjAm gCnANf"
        >
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            Download the Desktop App
          </div>
        </button>
        <div
          className="sc-AxjAm bgAmkT"
        >
          <div
            className="sc-AxjAm bgAmkT"
          >
            <test-file-stub
              height="100%"
              width="auto"
            />
          </div>
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            Get your information out of other people's hands and back into yours
          </div>
        </div>
        <div
          className="sc-AxjAm bgAmkT"
        >
          <div
            className="sc-AxjAm bgAmkT"
          >
            <test-file-stub
              height="100%"
              width="auto"
            />
          </div>
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            Protect yourself from phishing attacks and malicious extensions
          </div>
        </div>
        <div
          className="sc-AxjAm bgAmkT"
        >
          <div
            className="sc-AxjAm bgAmkT"
          >
            <test-file-stub
              height="100%"
              width="auto"
            />
          </div>
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            Open-source, fully audited, and full verifiable.
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-AxjAm iknhDZ"
  >
    <div
      className="sc-AxjAm hPWGaW"
      width={
        Object {
          "lg": "70%",
          "md": "80%",
          "sm": "90%",
        }
      }
    >
      <div
        className="sc-AxjAm sc-AxirZ jcPbhY"
      >
        Testimonials
      </div>
    </div>
  </div>
  <div
    className="sc-AxjAm bgAmkT"
  >
    <div
      className="sc-AxjAm iWucJa"
      width={
        Object {
          "lg": "70%",
          "md": "80%",
          "sm": "90%",
        }
      }
    >
      <div
        className="sc-AxjAm sc-AxirZ jcPbhY"
      >
        <div
          className="sc-AxjAm bmHhiV"
        >
          <div
            className="sc-AxjAm bgAmkT"
          >
            <test-file-stub
              height="100%"
              width="auto"
            />
          </div>
        </div>
        <h2
          className="sc-AxjAm hbdlIJ"
          fontFamily="body"
        >
          Ready to start managing your funds safely and confidently?
        </h2>
        <button
          className="sc-AxjAm gCnANf"
        >
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            Get Started
          </div>
        </button>
        <a
          className="css-vurnku"
          href="https://support.mycrypto.com/"
          rel="noreferrer"
          target="_blank"
        >
          <div
            className="sc-AxjAm bgQMNa"
            fontFamily="body"
          >
            Have Questions? We're Here to Help!
          </div>
        </a>
        <div
          className="sc-AxjAm kNlqYC"
          display={
            Object {
              "sm": "none",
              "xs": "block",
            }
          }
        >
          <div
            className="sc-AxjAm sc-AxirZ cFGRUg"
          >
            <div
              className="sc-AxjAm bgAmkT"
            >
              <test-file-stub
                height="100%"
                width="auto"
              />
            </div>
            <div
              className="sc-AxjAm bgAmkT"
            >
              <test-file-stub
                height="100%"
                width="auto"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
